From 7ea34c7bd49c6f72363ebaacbbaa2ce8293bc086 Mon Sep 17 00:00:00 2001
From: Kasual <mushroomhead52e@gmail.com>
Date: Sat, 21 Oct 2017 08:14:06 -0400
Subject: [PATCH 022/249] wiki: ensure users have a valid contributor agreement

Provide details on signing the contributor agreement so that new
users are aware it is a requirement prior to submitting changes

Change-Id: I22a2824b5197e9f2fc4a5b258c38bdb6e5e9f46d
---
 pages/how-tos/submitting_to_gerrit.md | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/pages/how-tos/submitting_to_gerrit.md b/pages/how-tos/submitting_to_gerrit.md
index 2430c38..61741b1 100644
--- a/pages/how-tos/submitting_to_gerrit.md
+++ b/pages/how-tos/submitting_to_gerrit.md
@@ -12,8 +12,11 @@ tags:
 
 If you haven't yet successfully downloaded the source and generated a build of LineageOS, make sure you are familiar with those steps. Information on doing a build is available in the build guide for [your device]({{ "/devices" | relative_url }}).
 
-Setup an account on [Gerrit](https://review.lineageos.org) and configure your Gerrit username in the Gerrit portal under **Settings -> HTTP Password**.
-Then, make sure your local git username matches with your Gerrit username::
+Setup an account on [Gerrit](https://review.lineageos.org), sign the [Contributor Agreement](https://review.lineageos.org/#/settings/agreements) and configure your Gerrit username in the Gerrit portal under **Settings -> HTTP Password**.
+
+{% include important.html content="Gerrit ensures users have completed a valid Contributor Agreement prior to accepting any transferred objects, and if it is not completed, it aborts the network connection before data is sent." %}
+
+Now make sure your local git username matches with your Gerrit username:
 
 ```
 git config --global review.review.lineageos.org.username "gerrit username"
-- 
2.17.1

